R/query_dataset.R

```diff
@@ -0,0 +1,57 @@
+#' Query a data set
+#'
+#' @description
+#' A short description...
+#'
+#' @inheritParams api_url
+#' @param method An API query method. Needs to be "GET"
+#'
+#' @return Data frame containing query results
+#' @export
+#'
+#' @examples
+#' query_dataset(example_id(), indicators = example_id("indicator"), page = 1, page_size=10)
```

Comment:
Also, if you do that currently with the test example you get this error (which I assume is a bug in the `get_dataset()` code?

```
query_dataset(example_id(), indicators = example_id("indicator"))
Error in rbind(deparse.level, ...) : 
  numbers of columns of arguments do not match
In addition: Warning message:
In query_dataset(example_id(), indicators = example_id("indicator")) :
  Using GET to query a data set offers limited functionality, we recommend using POST alongside a json structured query instead:
   - query_dataset(..., method = 'POST')
  ```

Reply:
Yep, that was the one that I clocked at Friday teatime. Fairly simple fix, just switched to dplyr::bind_rows() to allow for unmatched geography columns between pages.